onnxruntime/core/providers/webnn/builders/helper.h

```diff
@@ -249,14 +244,7 @@ inline bool CheckSingleOp(const std::string& op_type, const emscripten::val& wnn
     return false;
   }
   // Returns false if the WebNN op has not been implemented in MLGraphBuilder in current browser.
-  if (!wnn_builder[op_map.find(op_type)->second.opName].as<bool>()) {
-    return false;
-  }
-  // The current WebNN CPU (TFLite) backend supports a limited op list, and we'd rather
-  // fall back early to the ORT CPU EP rather than fail in the WebNN "cpu" deviceType.
-  // This is a workaround because the op may be included in MLGraphBuilder for DirectML
-  // backend but without TFLite implementation in Chromium.
-  if (!op_map.find(op_type)->second.isCpuSupported && device_type == WebnnDeviceType::CPU) {
+  if (!wnn_builder[op_map.find(op_type)->second].as<bool>()) {
```

Comment:
Hmm, what does `InlinedHashMap` return if the key is missing? Is it like std::map, and if so, is `->second` valid if you pass say `op_type == "foo"`? Should it be this?

```
  auto op_map_entry = op_map.find(op_type);
  if (op_map_entry == op_map.end() || !wnn_builder[op_map_entry->second].as<bool>()) {
```

Reply:
```
  if (op_map.find(op_type) == op_map.end()) {
    return false;
  }
```
We've check if it contains the key at previous code.

But your code looks more concise, I will use it. :)

Reply:
Fixed, PTAL again.

Reply:
> We've check if it contains the key at previous code

The previous code that was deleted? 😉

Reply:
Ah, I missed it in the folded code.